great-idea/services.html

```diff
@@ -4,7 +4,7 @@
 <head>
```

Comment:
Stretch needs some work.

Try looking up how to remove css style from buttons, or change them into text with a border.

"Services" should be bigger size.

Don't use "hr", it stretches across the entire page. Instead put a border-top or border-bottom on one of your divs.

Add a bit of margin/padding to the top of your 2nd image.

Try and get the bullet points aligned with the text above it.